synfig-studio/src/gui/widgets/widget_vector.cpp

```diff
@@ -61,13 +61,8 @@ Widget_Vector::Widget_Vector():
 	x_adjustment(Gtk::Adjustment::create(0,-100000000,100000000,0.05,0.05,0)),
 	y_adjustment(Gtk::Adjustment::create(0,-100000000,100000000,0.05,0.05,0))
 {
-	Gtk::Label *label;
 	int width_chars = 5;
 
-	label=manage(new class Gtk::Label("X:"));
-	label->set_alignment(0, 0.5);
-	//pack_start(*label, Gtk::PACK_SHRINK);
-	
 	entry_x=manage(new class Gtk::Entry());
```

Comment:
@morevnaproject Any chance to use labels 'X:' and 'Y:' in Widget_Vector in future?

Reply:
I can add a setting method to show/hide X-Y labels.
But it's commented since 7d21e999df 2015-03-17 .

Reply:
2015 was not so far)
np, we can remove these labels